src/PresentationalComponents/ExecutiveReport/Build.js

```diff
@@ -0,0 +1,91 @@
+/* eslint-disable camelcase */
+import { Battery, Chart, Column, Panel, PanelItem, Paragraph, Section, Table } from '@redhat-cloud-services/frontend-components-pdf-generator';
+
+import PropTypes from 'prop-types';
+import React from 'react';
+import { TOTAL_RISK_LABEL } from '../../AppConstants';
+import { Text } from '@react-pdf/renderer';
+import messages from '../../Messages';
+
+const BuildExecReport = ({ statsSystems, statsReports, topActiveRec, intl }) => {
+    const batteryMap = {
+        1: 'low',
+        2: 'medium',
+        3: 'high',
+        4: 'critical'
+    };
+    const severityPie = [
+        { x: intl.formatMessage(messages.critical), y: statsReports.total_risk[4] },
+        { x: intl.formatMessage(messages.important), y: statsReports.total_risk[3] },
+        { x: intl.formatMessage(messages.moderate), y: statsReports.total_risk[2] },
+        { x: intl.formatMessage(messages.low), y: statsReports.total_risk[1] }];
+    const severityRows = Object.entries(statsReports.total_risk).map((value) =>
+        [TOTAL_RISK_LABEL[value[0]].props.children, intl.formatMessage(messages.recNumAndPercentage, {
+            count: value[1],
+            total: Math.round(Number(value[1] / statsReports.total * 100))
+        })]);
+    severityRows.push([intl.formatMessage(messages.severity), intl.formatMessage(messages.numberRuleHits)]);
```

Comment:
No need to use push
```JS
const severityRows = [
  ...Object.entries(statsReports.total_risk).map(([ key, value ]) =>
        [TOTAL_RISK_LABEL[key].props.children, intl.formatMessage(messages.recNumAndPercentage, {
            count: value,
            total: Math.round(Number(value / statsReports.total * 100))
        })]),
  intl.formatMessage(messages.severity), intl.formatMessage(messages.numberRuleHits)
];
```

Reply:
omg, n i can throw it (the header array) in the front and get rid of the reverse, ahhh a perfect use of spread!